src/test/unit/event/test_event_dumper.py

```diff
@@ -483,6 +484,15 @@ def test_parse_cloudfront_event(cloudfront_event):
     )
 
 
+def test_event_size_insnt_trancated_when_using_extension(monkeypatch):
+    monkeypatch.setattr(os, "environ", {"LUMIGO_USE_TRACER_EXTENSION": "TRUE"})
+    event = {
+        "a": 'a' * 1000_000
```

Comment:
please use `Configuration.get_max_entry_size(True) * 2` (to ensure that the code actually been called)